app/models/solidus_acima/payment_method.rb

```diff
@@ -4,7 +4,6 @@ module SolidusAcima
   class PaymentMethod < SolidusSupport.payment_method_parent_class
```

Comment:
The title of the [commit](https://github.com/nebulab/solidus_acima/pull/6/commits/1454c9c797da67da2eb76d5f20972bb890692d1c) doesn't reflect what the code does.
The column is removed in the next commit.